docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift cards are available in Saleor 3.1.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiry dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create gift cards?
+Go to the Gift Card page of the Catalog section.
+
+:::note
+If you would like a gift card to act as a payment method, first you need to create a gift card [product type](dashboard/configuration/products.md).
+:::
+
+![Gift cards](../screenshots/catalog-giftcards-main.png)
+
+Then click the Issue Card button on the top right corner of the page. A pop-up page will appear where you can carry out the following actions related to the gift card creation:  
+
+- Choose currency  
+
+- Enter the amount  
+
+- Send gift cards to customer  
+
+- Set expiry date  
+
+- Write notes
+
+- Assign activation requirement
+
+![Gift cards](../screenshots/catalog-giftcards-new.png)
+
+You can also create multiple gift cards at the same time by clicking the three dots button on the top right corner of the page, near the Issue Card button. Then click the *Bulk Issue* button.
+
+![Gift cards-bulk issue](../screenshots/catalog-giftcards-bulk-issue.png)
+
+Once a gift card has been created you can manage its balance on the top of the gift card details page. You can also add multiple tags (optional) and set an expiration date.  
+
+On the right side of the gift card details page, you can deactivate a gift card, resend a code, and find basic information about the gift card such as:
+
+- Creation date
+
+- Link to order number to which gift card has been used
```

Comment:
```suggestion
- Link to order in which gift card has been used
```

Reply:
Double checked it. They are pretty much interchangeable here. I feel more comfortable with the first version.

Reply:
We're linking to an Order not an order number.